### DISCUSSION TITLE
when click  a .exe file  show error message

### DISCUSSION THREAD

**Discussion by wimotek** [spencerwooo/onedrive-vercel-index#277] [Q&A] [answered]
Oops, that's a four-oh-four.
Preview for file *******.exe is not available, download directly with the button below.

**Comment by spencerwooo** [comment]
What did you expect? You cannot preview executable files.

**Reply by wimotek** [reply]
i know i cannot preview exe file .    i hope  no this prompt mesage .     the preview windows is specified for docs pictures videos etc.

**Comment by spencerwooo** [comment, marked as answer]
unsupported or unknown file previews are shown as:

![image](https://user-images.githubusercontent.com/32114380/149659706-d6477a5f-a94f-43e6-ad6a-2978d8676979.png)

in the latest update
